templates/contribute/index.md

```diff
@@ -146,11 +146,11 @@ whether to proceed at all.
 
 ### Dealing with merge conflicts
 
-Due to the fact that multiple people work on mathlib in parallel, someone might have introduced a change on `master` that conflicts with a change that you're proposing on your PR. If it happens with your PR, check [this GitHub tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) on how to resolve merge conflicts by using their online tool.
+Due to the fact that multiple people work on mathlib4 in parallel, someone might have introduced a change on `master` that conflicts with a change that you're proposing on your PR. If it happens with your PR, check [this GitHub tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) on how to resolve merge conflicts by using their online tool.
 
 ## Caching compilation
 
-In the `mathlib` git repository, you can run the following in a terminal:
+In the `mathlib4` git repository, you can run the following in a terminal:
```

Comment:
These instructions are no longer needed, right?

Reply:
Yes, any mention of mathlibtools or leanproject should be removed.

Reply:
Removed mention of mathlibtools.